tests/test_show_users.py

```diff
@@ -0,0 +1,20 @@
+import snowflake.connector.cursor
+
+from fakesnow.connection import USERS_TABLE_FQ_NAME
+
+
+def test_show_users_base_case(cur: snowflake.connector.cursor.SnowflakeCursor):
+    result = cur.execute("SHOW USERS")
+    assert result
+    assert result.fetchall() == []
+
+
+def test_show_users_with_users(cur: snowflake.connector.cursor.SnowflakeCursor):
+    cur.execute(f"INSERT INTO {USERS_TABLE_FQ_NAME} (name) VALUES ('foo'), ('bar')")
```

Comment:
One of goals of the tests is that they pass on a real Snowflake instance (see [CONTRIBUTING.MD](https://github.com/tekumara/fakesnow/blob/main/CONTRIBUTING.md#raising-a-pr)). The tests also document the behaviour end users can expect will be stable, and I'd prefer not to guarantee an implementation detail like an internal database or table name won't change.

Ideally this would use `CREATE USER` which would satisfy the criteria above. But failing that, could we remove this test?

Reply:
It seemed important to have a test that asserted...anything about the tables existing columns or otherwise the result not being empty.